packages/preview-middleware/README.md

```diff
@@ -13,6 +13,7 @@ It hosts a local Fiori launchpad based on your configuration as well as offers a
 | `flp.intent.action`    | `string`  | `preview`        | Optional intent action                                                                                                              |
 | `flp.apps`             | `array`   | `undefined`      | Optional additional local apps that are available in local Fiori launchpad                                                          |
 | `flp.libs`             | `boolean` | `undefined`      | Optional flag to add a generic script fetching the paths of used libraries not available in UI5. To disable set it to `false`, if not set, then the project is checked for a `load-reuse-libs` script and if available the libraries are fetched as well. |
+| `flp.ui5Theme`             | `string` | `undefined`      | Optional flag for setting the UI5 Theme. |
```

Comment:
Could we just use `theme`, it should be clear that it is a UI5 theme.

Reply:
Sure, will rename

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/1311/commits/2eb13f05d63d1a520bd2ce800cd6f457ea93c9f2 and https://github.com/SAP/open-ux-tools/pull/1311/commits/b0064ff05a22aa46ffc527e6995e71c58fc93e43